src/assets/scripts/aircraft/AircraftConflict.js

```diff
@@ -0,0 +1,268 @@
+import Fiber from 'fiber';
+import { vlen, vsub } from '../math/vector';
```

Comment:
yeah, it's much cleaner.  It will take some time to move everything over to this style.  So some of that stuff will have to wait until the heavy lifting is out of the way.